src/ColorPicker/index.jsx

```diff
@@ -66,18 +66,20 @@ function ColorPicker({
           style={{ textAlign: 'start' }}
         >
           <HexColorPicker color={color || ''} onChange={setColor} />
-          <Form.Group className="pgn__hex-form" size="sm">
-            <h5>Hex</h5>
-            <Form.Control
-              className="form-field"
-              isInvalid={!hexValid}
-              value={color}
-              onChange={(e) => validateHex(e.target.value)}
-              data-testid="hex-input"
-            />
+          <Form.Group className="pgn__hex-form flex-wrap" size="sm">
```

Comment:
Paragon components cannot use CSS utility classes on them because they include `!important`. It makes it challenges to theme the component for brand authors. Would recommend putting `flex-wrap` equivalent in the CSS class name definition for `.pgn__hex-form`. 

Similar comments for all other CSS utility classes below, e.g. `d-inline-flex`, `flex-fill`, and `ml-2`.